elixir/apps/web/lib/web/live/sites/show.ex

```diff
@@ -264,7 +264,7 @@ defmodule Web.Sites.Show do
       <:action>
         <.delete_button
           phx-click="delete"
-          data-confirm="Are you sure want to delete this gateway group and disconnect all it's gateways?"
+          data-confirm="Are you sure want to delete this Site and disconnect all it's Gateways?"
```

Comment:
```suggestion
          data-confirm="Are you sure want to delete this Site and disconnect all its Gateways?"
```
grammar nit

Reply:
Thanks for catching that.  It made me go back and re-read the entire sentence and I realized there was a `you` missing as well.